source/scripts/app.js

```diff
@@ -85,16 +86,31 @@ function createCookbook() {
 function bindCreateCookbookSave() {
   "use strict";
   let shadow = document.querySelector("create-cookbook").shadowRoot;
-  let buttonHandler = shadow.getElementById("save-button");
-  buttonHandler.addEventListener("click", async () => {
+  let saveButton = shadow.getElementById("save-button");
+  let cancelButton = shadow.getElementById("cancel-button-container");
+
+  saveButton.addEventListener("click", async () => {
     let title = shadow.getElementById("title-input").value;
 
     if (title !== "") {
       let description = shadow.getElementById("description-input").value;
       await indexedDb.createCookbook(title, description);
+      populateCookbooksPage();
+
+      // Reset the page to original values
+      shadow.getElementById("title-input").value = null;
+      shadow.getElementById("description-input").value = null;
```

Comment:
Change the `null`s to `""` since that is what we are checking for in the `if` statement. The default `value` of the empty title input is also `""`. We should probably define a constant for `""` like `const NO_INPUT = "";` at the top of the file. Good catch resetting these values by the way.
```suggestion
      shadow.getElementById("title-input").value = "";
      shadow.getElementById("description-input").value = "";
```